lib/core/utils/filter-html-attrs.js

```diff
@@ -73,3 +62,18 @@ function filterHtmlAttrs(element, filterAttrs) {
 }
 
 export default filterHtmlAttrs;
```

Comment:
The default export function itself should be moved to the top with an inlined `default export`.